Resources/Prototypes/Catalog/VendingMachines/Inventories/seeds.yml

```
@@ -38,5 +38,6 @@
     BerrySeeds: 5
     PeaSeeds: 5
     CottonSeeds: 5
+    CoffeeSeeds: 5
```

Note that this has been added in Frontier's code.
```suggestion
    CoffeeSeeds: 5 # Frontier
```

---

Just to help me understand the two above: Thats a comment to mark it its sourced on our base not main ss14?

Other than leaving a mark, does that affect any other processes? Just curious.

---

Yup!  And not currently, but it's helpful in resolving merge conflicts.